Integrations/McAfee_DXL/McAfee_DXL.py

```diff
@@ -140,9 +164,8 @@ def main():
 
         return_outputs(result)
     except Exception as error:
+        validate_certificates_format()
         return_error(f'error in {INTEGRATION_NAME} {str(error)}.', error)
-    finally:
-        event_sender.client.disconnect()
```

Comment:
Why did you remove the `finally` clause?

Reply:
I consulted with @esharf (who wrote the integration) and we decided to move it to __del__ function of the EventSender class.